observability/otel/otel-collector/otel-collector-connectors.adoc

```diff
@@ -8,6 +8,108 @@ toc::[]
 
 A connector connects two pipelines. It consumes data as an exporter at the end of one pipeline and emits data as a receiver at the start of another pipeline. It can consume and emit data of the same or different data type. It can generate and emit data to summarize the consumed data, or it can merely replicate or route data.
 
+[id="count-connector_{context}"]
+== Count Connector
+
+The Count Connector counts trace spans, trace span events, metrics, metric data points, and log records in exporter pipelines.
+
+:FeatureName: The Count Connector
+include::snippets/technology-preview.adoc[]
+
+The following are the following default metric names:
```

Comment:
I think this would be easier for customers
```suggestion
The following are the default exposed metrics:
```

Reply:
Oops. Thanks for spotting this. Fixed.